src/hotspot/cpu/aarch64/compressedKlass_aarch64.cpp

```diff
@@ -129,5 +129,7 @@ void CompressedKlassPointers::initialize(address addr, size_t len) {
   address const end = addr + len;
   _base = (end <= (address)unscaled_max) ? nullptr : addr;
 
-  _range = end - _base;
+  // Remember the Klass range:
+  _klass_range_start = addr;
+  _klass_range_end = addr + len;
```

Comment:
In the Lilliput review, I think you were moving this to shared code with an ifdef so that these variables are all set in one place.  This is ok here for now.

Reply:
Yes I did this. I like the Lilliput variant better too.